### PR TITLE
Feature/CLX-486 loading error handling for termination flow

### PR DESCRIPTION
**Pull request by julia-andersson**
## [CLX-486]

- Added loading for termination flow screens

## Checklist

- [x] Dark/light mode verification
- [x] Landscape verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

[CLX-486]: https://hedvig.atlassian.net/browse/CLX-486?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Projects/Contracts/Sources/ContractAction.swift [hunk 1]
```diff
@@ -89,3 +89,9 @@ public enum TerminationNavigationAction: ActionProtocol {
     case openTerminationFailScreen
     case openTerminationDeletionScreen
 }
+
+extension ContractAction: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine("\(try! JSONEncoder().encode(self))")
```
Comment (sampettersson) [inline Projects/Contracts/Sources/ContractAction.swift:95]: If you define Hashable directly on the ContractAction and all the associated values are Hashable this shouldn't be necessary, it should do that automatically.

**Review by sampettersson — CHANGES_REQUESTED** [review]

File: Projects/Contracts/Sources/ContractAction.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Contracts/Sources/ContractAction.swift:95]: We had to do it since it was complaining. It does not make sense to me either.

Comment (sladan-hedvig) [reply Projects/Contracts/Sources/ContractAction.swift]: Done!

**Review by sampettersson — APPROVED** [review]
🚀

**Pull request merged**